domains/aniket-thapa.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "aniket-thapa",
+           "email": "aniketthapa04@gmail.com",
+           "discord": "1249367001961463962"
+        },
+    
+        "record": {
+            "CNAME": "portfolio.github.io"
```

Comment:
Site inaccessible.
I did notice that your site is hosted on aniket-thapa.github.io/portfolio, so I'll change it back to aniket-thapa.github.io since thats how you needed to do it